### PR TITLE
bind toggle handler

### PR DESCRIPTION
**Pull request by wheresrhys**
@AlbertoElias

### CODE REVIEW

File: src/js/utils.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Utils {
 		this.selectableHandler = this.selectableHandler.bind(this);
 		for (let selectableEl of selectableEls) {
 			this.listeners.push([selectableEl, this.selectableHandler]);
-			selectableEl.addEventListener('click', this.selectableHandler);
+			selectableEl.addEventListener('click', this.selectableHandler.bind(this));
```
Comment (AlbertoElias) [inline src/js/utils.js]: this is binded in line 22

File: src/js/utils.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ class Utils {
 		}
 
 		const toggleEls = this.headerEl.querySelectorAll('[data-o-header-togglable]');
+		this.toggleHandler = this.toggleHandler.bind(this);
```
Comment (AlbertoElias) [inline src/js/utils.js]: why does this need to be binded?

File: src/js/utils.js (hunk 1 above)
Comment (wheresrhys) [inline src/js/utils.js]: oops meant to remove before pushing

File: src/js/utils.js (hunk 2 above)
Comment (wheresrhys) [inline src/js/utils.js]: same reason all the other handlers need to

Comment (AlbertoElias) [inline src/js/utils.js]: Other handlers needed it because they needed to reference the utils instance, but the toggleHandler doesn't use this anywhere

Comment (wheresrhys) [inline src/js/utils.js]: this.headerEl

Comment (AlbertoElias) [inline src/js/utils.js]: What line?

**Pull request closed without merging**